src/Avalonia.X11/X11Platform.cs

```diff
@@ -289,6 +298,11 @@ public class X11PlatformOptions
         /// </remarks>
         public bool? EnableMultiTouch { get; set; } = true;
 
+        /// <summary>
+        /// Enables Vulkan rendering backend
+        /// </summary>
+        public bool UseVulkan { get; set; }
```

Comment:
This seems to  fit into what is already there. UseXy is present in all kinds of backends. I wonder whether at some point we do not distribute UseXy into all separate XyPlatformOptions. Anyways fits to how it is done already.